pootle/checks.py

```diff
@@ -267,11 +267,11 @@ def check_settings(app_configs=None, **kwargs):
         if markup_filter is not None:
             try:
                 if markup_filter == 'textile':
-                    import textile  # noqa
+                    import textile  # noqa pylint:disable=unused-variable
                 elif markup_filter == 'markdown':
-                    import markdown  # noqa
+                    import markdown  # noqa pylint:disable=unused-variable
                 elif markup_filter == 'restructuredtext':
-                    import docutils  # noqa
+                    import docutils  # noqa pylint:disable=unused-variable
```

Comment:
@dwaynebailey i reckon we could fix this relatively easily with importlib

heres a SO discussion about similar http://stackoverflow.com/questions/14050281/how-to-check-if-a-python-module-exists-without-importing-it